examples/with-nextjs-and-clerk-auth/src/lib/monite-api/demo-data-generator/bin.ts

```diff
@@ -46,6 +48,115 @@ const commandWithEntityOptions = () =>
     .option('--entity-user-id <ID>', 'The entity_user_id to use', String);
 
 program
+  .addCommand(
+    commandWithEntityOptions()
+      .name('run')
+      .description('Generate demo data for the existing Entity')
+      .action(async (args) => {
+        const token = await fetchTokenCLI(args);
+
+        const { id: newEntityId } = await createOrganizationEntity(
+          {
+            email: faker.internet.email(),
+            legal_name: faker.company.name(),
+          },
+          token
+        );
+
+        await generateDemoData(newEntityId, token);
+      })
+  )
+  .addCommand(
+    new Command()
+      .name('entity')
+      .description('Create a new Entity')
+      .option(
+        '--generate-demo-data',
+        'Generate demo data for the new entity',
+        true
+      )
+      .option('--client-id <ID>', 'The client_id to use', String)
+      .option('--client-secret <ID>', 'The client_secret to use', String)
+      .action(async (args) => {
+        const token = await fetchTokenCLI(args);
```

Comment:
What will happen if `await` throws an error? Does the `commander` library handle it and print errors in the console?

Shouldn't we wrap it into `try / catch` and in `catch` print an error into the console & to Clerk UI?